src/Service/CaptureService.php

```diff
@@ -173,7 +176,9 @@ public function doOpenInvoiceCapture(string $orderNumber, $captureAmount, Contex
                     }
                     $results[] = $response;
                 } else {
-                    throw new CaptureException('Order delivery status does not match configuration');
+                    $exception = new CaptureException('Order delivery status does not match configuration');
+                    $exception->reason = self::REASON_DELIVERY_STATE_MISMATCH;
+                    throw $exception;
```

Comment:
This scenario is not an error per se, this just means that the merchant needs to move update the delivery state before the capture request can be sent.
If we count this scenario as an error and increment the error count, then the notification will be lost after 3 trials and the payment will never be captured.